Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Subject: Tighten dependencies on base64

Debian build daemons and the rust packaging scheme will default to the highest
version available.  But debian only has the base64 crate at version 0.13
right now.

--- sequoia-net/Cargo.toml
+++ sequoia-net/Cargo.toml
@@ -49,7 +49,7 @@
 version = "1.0.18"
 
 [dependencies.base64]
-version = ">=0.12, <0.20"
+version = ">=0.12, <0.14"
 
 [dependencies.futures-util]
 version = "0.3.5"
